src/commands/FxTweet.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  CommandInteraction,
+  Client,
+  ApplicationCommandType,
+  ApplicationCommandOptionType,
+  InteractionResponse,
+} from 'discord.js';
+
+import { Command } from '../interfaces/Command';
+
+export const FxTweet: Command = {
```

Comment:
Up to you, but using `satisfies as Command` should provide more accurate typing when hovering the mouse over it:
```ts
const FxTweet: Command
// vs
const FxTweet: {
    name: string;
    description: string;
    type: ApplicationCommandType.ChatInput;
    options: {
        name: string;
        description: string;
        type: ApplicationCommandOptionType.String;
        required: true;
    }[];
    run: (_: Client<...>, interaction: CommandInteraction<...>) => Promise<...>;
}
```